template_app/src/routes/+page.svelte

```diff
@@ -8,11 +8,13 @@
   import TitleMode from "./TitleMode.svelte";
   import TriangleMode from "./TriangleMode.svelte";
   import { map as mapStore, backend, mode } from "./globals";
-  import workerWrapper from "./worker?worker";
-  import { type Backend } from "./worker";
   import * as Comlink from "comlink";
   import { onMount } from "svelte";
   import type { Map } from "maplibre-gl";
+  //import workerWrapper from "./rust_worker?worker";
```

Comment:
As the PR description says, this is an intermediate hack for now. I'll look into the CLI generator next and decide how to swap out the choice